course_access_groups/models.py

```diff
@@ -4,3 +4,154 @@
 """
 
 from __future__ import absolute_import, unicode_literals
+
+from django.contrib.auth.models import User
+from django.core import exceptions, validators
+from django.db import models
+from django.utils.text import slugify
+from django.utils.translation import ugettext as _
+from model_utils import models as utils_models
+from organizations.models import Organization
+from six import text_type
+
+
+def validate_domain(value):
```

Comment:
Is it necessary to include this function in the `models` module?

Reply:
Nope, moved to its own `validators` module.

Reply:
Nope, moved to its own `validators` module.